homework/index.js

```diff
@@ -30,13 +30,127 @@
     return elem;
   }
 
+  function buildRepositoryInfo(repositoryObject, containerDiv) {
+    const repositoryTable = createAndAppend('table', containerDiv, {
+      id: 'repository-table',
+    });
+    const repositoryTableBody = createAndAppend('tbody', repositoryTable, {});
+    const repositoryTableRow1 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow1, { text: 'Repository:' });
+    const tableName = createAndAppend('td', repositoryTableRow1, {});
+    createAndAppend('a', tableName, {
+      text: repositoryObject.name,
+      href: repositoryObject.html_url,
+      target: '_blank',
+    });
+    const repositoryTableRow2 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow2, { text: 'Description:' });
+    createAndAppend('td', repositoryTableRow2, { text: repositoryObject.description });
+    const repositoryTableRow3 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow3, { text: 'Forks:' });
+    createAndAppend('td', repositoryTableRow3, { text: repositoryObject.forks_count });
+    const repositoryTableRow4 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow4, { text: 'Last updated:' });
+    createAndAppend('td', repositoryTableRow4, { text: repositoryObject.updated_at });
+  }
+
+  function buildRepositoryContributors(contributorsArray, containerDiv) {
+    createAndAppend('h2', containerDiv, { text: 'Contributions' });
+    const contributorsList = createAndAppend('ul', containerDiv, { id: 'contributors-list' });
+    contributorsArray.forEach(element => {
```

Comment:
If the function argument was called `repositories` then the `forEach` loop could be written as:

```js
contributors.forEach(contributor => {
```

Try and avoid generic names such as `item`, `element` if you have precise knowledge what the variable or parameter contains.